config/locales/en.yml

```diff
@@ -191,26 +191,29 @@ toplevel:
     asideAnnualReport:
         quoteText: In 2016/17 we awarded over £700 million, including 10,000 grants under £10,000
         linkText: Read our latest annual report
-        linkHref: #
+        linkHref: "/about-big/publications/corporate-documents"
         imageCaption: Pennine Lancashire Community Farm
-    goodCauses: The National Lottery raises money for good causes. It makes a difference to people by
-      transforming communities, protecting our heritage, and enriching lives through arts, sports and culture.
-      Communities come in all shapes and sizes, and National Lottery funding is there for everyone.
+    howOurFundingIsUsed:
+        part1: People use our funding to do extraordinary things,
+          taking the lead to improve their lives and communities.
+          Every time you buy a National Lottery ticket, you help make this happen.
+        part2: The National Lottery raises money for good causes.
+          It makes a difference to people by transforming communities, protecting our heritage,
+          and enriching lives through arts, sports and culture. Communities come in all shapes and sizes,
+          and National Lottery funding is there for everyone.
     ourStructure:
         title: Our structure
-        leadership: We are led by our <a href="#">senior management team</a> and overseen by a <a href="#">board of directors</a>.
+        leadership: We are led by our <a href="/about-big/our-people/senior-management-team">senior management team</a> and overseen by a <a href="/about-big/our-people/board">board of directors</a>.
         governing: Learn about our structure and how we’re governed
-        established: We’re the largest distributor of National Lottery funding,
-          established as a non-departmental public body by an Act of Parliament
     asidePeopleInTheLead:
         quoteText: We reached nearly 10 million people with our funding last year
         linkText: "Discover our strategic framework: People in the Lead"
-        linkHref: #
+        linkHref: "/about-big/strategic-framework"
         imageCaption: Dawn Austwick, CEO
-    outro: We are the largest distributor of National Lottery funding,
-      established as a non-departmental public body by an Act of Parliament.
-      Our work is divided into five portfolios, which cover funding across England,
-      Northern Ireland, Wales and Scotland, and the UK as a whole.
+    outro: We’re the largest distributor of National Lottery funding,
+      established as a non- departmental public body by an Act of Parliament.
+      Our work is divided into five major portfolios, which covering funding across
```

Comment:
Should this be "cover"?

Reply:
Just checked the doc, and I think so. I must've picked up some track-changes leftovers.